ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/ValueLimitsIT.scala

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import com.daml.ledger.api.testtool.infrastructure.Allocation._
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.api.testtool.infrastructure.ProtobufConverters._
+import com.daml.ledger.test.model.Test.DummyWithAnnotation
+
+import scala.concurrent.Future
+import scala.concurrent.duration.{DurationInt, FiniteDuration}
+import scala.util.Random
+
+final class ValueLimitsIT(timeoutScaleFactor: Double) extends LedgerTestSuite {
+
+  /** Postgres has a limit on the index row size of 2712.
+    * We test that a large submitters number doesn't cause deduplication failure because of that limit.
+    * THIS TEST CASE DOES NOT TEST ANY OTHER FAILURES THAT MAY BE CAUSED BY A LARGE SUBMITTERS NUMBER
+    */
```

Comment:
This description is not accurate:
- The test uses the ledger API and not the internals of the indexer,  so you don't really know what kind of failures it tests for.
- The description mentions a limit of Postgres, but you don't know whether the test runs against a Postgres index database.
- You are not checking any property of command deduplication, you are just submitting a request with a large number of parties. I would name the test `"Submit commands with a large number of submitters"`.

Reply:
@rautenrieth-da Thank you for the review!
I've modified the test suite according to your suggestions. I've also added a change to a database index that was blocking submissions due to index row size limits.